app/build.gradle

```diff
@@ -91,8 +91,11 @@ dependencies {
     def arklibVersion = '0.3.2'
 
     releaseImplementation "dev.arkbuilders:arklib:$arklibVersion"
-    debugImplementation "dev.arkbuilders:arklib-debug:$arklibVersion"
-    implementation("dev.arkbuilders:components:0.0.3") {
+    debugImplementation ("dev.arkbuilders:arklib-debug:$arklibVersion") {
+        //REMOVE below line after upgrading filepicker to latest version in arklib/arklib-debug
+        exclude group: "com.github.ARK-Builders", module: "ark-filepicker"
+    }
+    implementation("dev.arkbuilders:components:0.0.4-snapshot-01") {
```

Comment:
Change to new `components` version after releasing.

Reply:
Should be v0.0.6

Reply:
https://github.com/ARK-Builders/ark-components-android/packages/1930514

Reply:
@kirillt Done upgrading to v0.0.6.